SampleApplications/Samples/Client.Net4/Opc.Ua.SampleClient.Config.xml

```diff
@@ -174,11 +174,11 @@
         <SecurityPolicyUri>http://opcfoundation.org/UA/SecurityPolicy#Basic256</SecurityPolicyUri>
         <SecurityLevel>2</SecurityLevel>
       </ServerSecurityPolicy>
-      <ServerSecurityPolicy>
+      <!--<ServerSecurityPolicy>
```

Comment:
A comment why you have commented this out can help folks looking at this code to understand the reason, without looking up the history.